src/pages/uxp-api/reference-js/Global Members/HTML DOM/Element.md

```diff
@@ -317,6 +317,48 @@ Returns the open shadow root that is hosted by the element, or null if no open s
 **Read only**
 
 
+<a name="element-append" id="element-append"></a>
+
+## append(...nodes)
+Inserts a set of Node objects or string objects after the last child of the Element.
+
+**See**: https://developer.mozilla.org/en-US/docs/Web/API/Element/append  
+**Since**: v8.0  
+
+| Param | Type |
+| --- | --- |
+| ...nodes | `Array<Node>` | 
+
+
+
+<a name="element-prepend" id="element-prepend"></a>
+
+## prepend(...nodes)
+Inserts a set of Node objects or string objects before the first child of the Element.
+
+**See**: https://developer.mozilla.org/en-US/docs/Web/API/Element/prepend  
+**Since**: v8.0  
+
+| Param | Type |
+| --- | --- |
+| ...nodes | `Array<Node>` | 
+
+
+
+<a name="element-replacechildren" id="element-replacechildren"></a>
+
+## replaceChildren(...nodes)
```

Comment:
Is it replaceChildren or replaceWith?

Reply:
It is `replaceChildren`
[ref](https://git.corp.adobe.com/torq/torq-native/blob/d828161a8816d0bbd66fd149d497f7122d7f48e5/modules/bindings/domjs/src/js/dom/nodes/Element.js#L338)

Reply:
The Changelog3P says replaceWith.